docs/admin/config.rst

```diff
@@ -61,7 +61,7 @@ Maximum number of failed authentication attempts before rate limiting is applied
 
 This is currently applied in the following locations:
 
-* Logins. Deletes the account password, preventing the user from login in
+* Logins. Deletes the account password, preventing the user from logging in
```

Comment:
```suggestion
* Logins. Deletes the account password, preventing the user from signing in
```

We use "sign in" in the UI, so it probably should be consistent in the docs as well.